packages/fxa-content-server/app/scripts/lib/fxa-client.js

```diff
@@ -1284,6 +1284,15 @@ FxaClientWrapper.prototype = {
    */
   getActiveSubscriptions: createClientDelegate('getActiveSubscriptions'),
 
+  /**
+   * Get a list of settings data with a session token.
+   *
+   * @param {String} token A token from the OAuth server.
+   * @returns {Promise} A promise that will be fulfilled with a list of settings
+   * data.
+   */
+  account: createClientDelegate('account'),
```

Comment:
@LZoog, not sure how close this PR is to landing but just fyi, I've copied some of this stuff into the branch that @meandavejustice started for #970. Whoever lands second might get some merge conflicts, but they shouldn't be anything too complicated!